drivers/dma/dma_stm32.c

```diff
@@ -629,12 +632,14 @@ static void dma_stm32_config_irq_1(struct device *dev)
 	IRQ_INIT(1, 4);
 #ifdef DT_INST_1_ST_STM32_DMA_IRQ_5
 	IRQ_INIT(1, 5);
+#ifdef DT_INST_1_ST_STM32_DMA_IRQ_6
 	IRQ_INIT(1, 6);
 #ifdef DT_INST_1_ST_STM32_DMA_IRQ_7
 	IRQ_INIT(1, 7);
-#endif
-#endif
-#endif
-/* Either 0 or 5 or 7 or 8 channels for DMA1 across all stm32 series. */
+#endif /* DT_INST_1_ST_STM32_DMA_IRQ_5 */
+#endif /* DT_INST_1_ST_STM32_DMA_IRQ_6 */
+#endif /* DT_INST_1_ST_STM32_DMA_IRQ_7 */
+/* Either 0 or 5 or 6 or 7 or 8 channels for DMA2 across all stm32 series. */
```

Comment:
Since instances are permutable, we should support the same #idef scheme on both instances

Reply:
ok, updating comment